deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html

```diff
@@ -47,7 +47,7 @@
 
     <p>
         In accordance with the usage terms of Developer Sandbox, we have reduced the number of instances of your
-        application to zero (0). You can restart your application(s) by increasing the number of instances from the
+        application {{.PodName}} to zero (0). You can restart your application(s) by increasing the number of instances from the
```

Comment:
just a small reminder - based on https://github.com/codeready-toolchain/member-operator/pull/478#discussion_r1346958235 should we rename the variable to `.AppName` ?

Reply:
will be changing it applicationname